### DISCUSSION TITLE
Follow-on to SourceForge -- Tonic vs Phasic Activity in VR

### DISCUSSION THREAD

**Discussion by msloan31** [bachlab/PsPM#469] [Q&A] [answered]
Thank you for the response in SourceForge (follow-up discussion here):  summarizing my question:  I have collected skin conductance data on 975 VR video experiences that are 1-minute each.  I treat each scene as one event, so I am looking for sympathetic skin response throughout the whole of the video (an urban environment), without identifying specific events.  The goal is to see the effect of each urban space on physiological restoration (in conjunction with other physiological and psychological measures).  I am trying to determine which model might be more appropriate, based on GLM or DCM.  The reply asked if I am more interested in tonic arousal (favoring SF-DCM) or event-related arousal.  I seek to compare the overall phasic activity for each VR experience.  Would this favor a GLM model?  Would there also be a benefit to measure the tonic activity, if this could show a more parasympathetic response?

**Comment by dominikbach** [comment, marked as answer]
Hi Mark. If you are interested in phasic activity over a 1-min period (which I would have termed this "tonic arousal", but that's just a question of definitions) then I'd suggest SF-DCM (or as a fast approximation, SF-MP). I don't think there is any direct comparison of this method with the ones implemented in Ledalab for that particular use case. Hope this helps, Dominik
